src/steps/custom-object-create-or-update.ts

```diff
@@ -0,0 +1,96 @@
+/*tslint:disable:no-else-after-return*/
+
+import { BaseStep, Field, StepInterface } from '../core/base-step';
+import { Step, FieldDefinition, StepDefinition } from '../proto/cog_pb';
+
+export class CreateOrUpdateCustomObjectStep extends BaseStep implements StepInterface {
+
+  protected stepName: string = 'Create or Update a Marketo Custom Object';
+  protected stepExpression: string = 'create or update an? (?<name>.+) marketo custom object linked to lead (?<linkValue>.+)';
+  protected stepType: StepDefinition.Type = StepDefinition.Type.ACTION;
+  protected expectedFields: Field[] = [{
+    field: 'name',
+    type: FieldDefinition.Type.STRING,
+    description: "Custom Object's display or API name",
```

Comment:
I noticed in the code below that you are directly using `stepData.name` to get the definition of the custom object (e.g. in the URL string of the API endpoint), which would not work in the case of a custom object's display name.

Upon reflection, I think this is sufficient for now; we require users to use field API names (rather than labels), so this seems like a reasonable limitation, and it saves us an API call.

With that in mind:

- [ ] Could you modify this label to just read `Custom Object's API name` and remove `display`